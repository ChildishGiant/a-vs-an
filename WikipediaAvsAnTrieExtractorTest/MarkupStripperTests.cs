using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using ExpressionToCodeLib;
using WikipediaAvsAnTrieExtractor;
using Xunit;

namespace WikipediaAvsAnTrieExtractorTest {
    
    public class MarkupStripperTests {
        readonly RegexTextUtils utils = new RegexTextUtils();

        [Fact]
        public void StripsMarkup()
        {
            var data = new Dictionary<string, string>
            {
            };
            foreach (var kvp in data) {
                PAssert.That(() => kvp.Value == utils.StripWikiMarkup(kvp.Key));
            }
        }
    }
}
