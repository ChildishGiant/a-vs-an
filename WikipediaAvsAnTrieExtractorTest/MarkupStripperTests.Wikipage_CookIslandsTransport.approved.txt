

This article lists transport in the Cook Islands.

; Railways:
: 0.1 km (Rarotonga Steam Railway)
; Highways:
:* Total: 320 km
:* Paved: 33 km
:* Unpaved: 287 km (2003 est.)
; Ports and harbours:
: Avarua, Avatiu
; Merchant marine:
:* Total: 1 ship (1,000 GRT or over) totalling 2,310 GRT/
:* Ships by type: Cargo 1 (1999 est.)
; Airports:
: 7 (1999 est.)
; Airports - with paved runways:
:* Total: 2
:* 1,524 to 2,437 m: 1 (1999 est.)
; Airports - with unpaved runways:
:* Total: 6
:* 1,524 to 2,437 m: 3
:* 914 to 1,523 m: 3 (1999 est.)

 See also ==
 Cook Islands

External links==
 Airports and Ports in the Cook Islands






